crates/contract/src/event.rs

```diff
@@ -154,7 +220,7 @@ mod tests {
     #[tokio::test]
     async fn event_filters() {
         init_tracing();
-        let (provider, _anvil) = spawn_anvil();
+        let (provider, anvil) = spawn_anvil();
```

Comment:
Keep underscore for `not(feature = "pubsub")`

```suggestion
        let (provider, _anvil) = spawn_anvil();
```